src/multiplayer-servers/multiplayer-services/game-server-wrapper.md

```diff
@@ -0,0 +1,217 @@
+|----------------------------------------|:--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <code v-pre>{{.GameServerIP}}</code>   | `string`            | The IP given by Agones to the game server.                                                                                                                                                                  |
+| <code v-pre>{{.GameServerPort}}</code> | `uint16`            | The port given by Agones to the game server. Defaults to the port named `game`, or if there is no port with that name, to the first port found. A different name can be configured using `--ports.primary`. |
+| <code v-pre>{{.Ports}}</code>          | `map[string]uint16` | The known game server ports.                                                                                                                                                                                |
+| <code v-pre>{{.Env}}</code>            | `map[string]string` | The environment variables.                                                                                                                                                                                  |
+
+**Example:**
+
+```shell
+/app/gsw -- /app/gameserver --port={{ .GameServerPort }} --query-port={{ .Ports.query }} --servername={{ .Env.POD_NAME }}
+```
+
+#### Configuration Files
+
+The wrapper can be configured to generate a template configuration file before invoking the game server.
+It needs a template file as input, which must be available in the wrapper's container, and an output file path.
+The available template variables are the same as for the [command-line arguments](#command-line-arguments).
+
+| Command-line argument    | Environment variable   | Description                                                |
+|--------------------------|------------------------|------------------------------------------------------------|
+| `--config.template-path` | `CONFIG_TEMPLATE_PATH` | Path at which to write the game server configuration file. |
+| `--config.output-path`   | `CONFIG_OUTPUT_PATH`   | Path to the configuration file template.                   |
+
+**Example:**
+
+```yaml
+# template.yaml
+gameserver:
+  ip: "{{ .GameServerIP }}"
+  port: "{{ .GameServerPort }}"
+  {{ - if .Env.POD_NAME }}
+  servername: "{{ .Env.POD_NAME }}"
+  {{ - end }}
+```
+
+```shell
+gsw --config.template-path=template.yaml --config.output-path=config.yaml -- /app/gameserver --config=config.yaml
+```
+
+### Log tailing
+
+The wrapper supports tailing log files and printing them to stdout using the wrapper's logger.
```

Comment:
```suggestion
The wrapper supports tailing log files and printing them to stdout using the wrapper's logger. This can be used to enable log collection for log files, which would otherwise be inaccessible.
```